### DISCUSSION TITLE
Hello. Can I add a custom page to the menu?

### DISCUSSION THREAD

**Discussion by masksshow** [aminalaee/sqladmin#236] [Q&A] [answered]
Hello.
Can I add a custom page to the menu? It is not related to the model.
For example, a statistics page or a custom template.

**Comment by aminalaee** [comment]
Hey, no unfortunately that's not possible yet.
Feel free to create a PR for it though.

**Comment by aminalaee** [comment, marked as answer]
This is now possible @masksshow https://aminalaee.dev/sqladmin/api_reference/base_view/
I will work on the docs for this

**Comment by masksshow** [comment]
You are the coolest!!!
Every day I pray for the addition of this. =)
https://github.com/aminalaee/sqladmin/discussions/272

**Reply by aminalaee** [reply]
Thanks @masksshow 😅 

Actually that's is very close, I needed some changes before doing that, now I can easily add it.
